contrib/kafka/Readme.md

```diff
@@ -0,0 +1,103 @@
+# Knative KafkaEventSource
+
+This deploys an Operator that can instantiate Knative Event Sources which subscribe to a Kafka Topic.
+
+## EventSource
+
+The image that receives messages from Kafka (Receive Adaptor) is available at `docker.io/sjwoodman/kafkaeventsource`.
+To build it from source edit the `eventsource/Makefile` with suitable tags for your DockerHub organisation and perform the following commands.
+
+```bash
+make docker_build docker_push_latest
+```
+
+You will need to edit the _Operator_ with the location of the newly built image.
+
+```bash
+sed -i 's|sjwoodman|DH_ORG|g' kafkaeventsource-operator/deploy/operator.yaml
+sed -i 's|sjwoodman|DH_ORG|g' kafkaeventsource-operator/pkg/controller/kafkaeventsource
```

Comment:
Path should be `kafkaeventsource-operator/pkg/controller/kafkaeventsource/kafkaeventsource_controller.go`.